src/app/screens/HomeScreen/index.tsx

```diff
@@ -18,6 +19,7 @@ function HomeScreen({ navigation }: Navigation<Book>) {
       data={DATA}
       renderItem={renderBook}
       keyExtractor={keyExtractor}
+      contentContainerStyle={NAV_MARGIN}
```

Comment:
Same here:
```
<SafeAreaView ...>
  <FlatList<Book>
...
```

Reply:
If I do that then the flatlist would start at the bottom of the nav bar and you would not see the cells passing behind the transparent nav bar. Also i have the size problem I mentioned in the scrollView